core/includes/install.core.inc

```diff
@@ -1409,28 +1383,51 @@ function install_select_language(&$install_state) {
  *
  * @ingroup forms
  */
-function install_select_language_form($form, &$form_state, $files) {
+function install_select_language_form($form, &$form_state, $files, $install_state) {
   include_once BACKDROP_ROOT . '/core/includes/standard.inc';
   include_once BACKDROP_ROOT . '/core/includes/locale.inc';
 
-  $standard_languages = standard_language_list();
+  $standard_languages = install_available_translations();
   $select_options = array();
   $languages = array();
 
-  foreach ($files as $file) {
-    if (isset($standard_languages[$file->langcode])) {
-      // Build a list of select list options based on files we found.
-      $select_options[$file->langcode] = $standard_languages[$file->langcode][1];
-    }
-    else {
-      // If the language was not found in standard.inc, display its langcode.
-      $select_options[$file->langcode] = $file->langcode;
+  if (count($install_state['translations']) > 1) {
+    $files = $install_state['translations'];
+  }
+  else {
+    $files = [];
+  }
+  $select_options = [];
+  $browser_options = [];
+
+  $form['#title'] = 'Choose language';
```

Comment:
This string needs to be translatable. I think `st()` should be used.

Reply:
No. At this point there can't be any translations, yet. We don't even have a database at this point. :wink:

Reply:
Fair enough. Can we then please add a comment indicating that fact?

Reply:
> No. At this point there can't be any translations, yet

I don't know if this is the reasoning here, since `st()` is used to `backdrop_set_title()` both times this function is called. I suspect its missing an `st()`.

Reply:
Added st()